spec/classroom_spec.rb

```diff
@@ -0,0 +1,9 @@
+$LOAD_PATH << '.'
+require 'spec_helper'
+
+describe Classroom do
+  it 'Create a new classroom' do
+    @new_class = Classroom.new('Ruby Class')
+    expect(@new_class.label).to eq('Ruby Class')
+  end
```

Comment:
- [ ] Same issue as above. Here we can do just one more thing, assert that we can add a student to a classroom and that will be all.